### PR TITLE
Added Firebase connector

### PR DESCRIPTION
**Pull request by pamelazagatti**

### CODE REVIEW

**Comment by pamelazagatti** [comment]
Hi @anna-tsulun , did you have a chance to review this? Is something missing or wrong?
Thank you

File: src/connector/FirebaseConnector.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+const Connector = require('./Connector');
+
+class FirebaseConnector extends Connector {
+    constructor(firebaseClient, rootRef) {
+        super();
```
Comment (anna-tsulun) [inline src/connector/FirebaseConnector.js:5]: Can you please add a comment explaining what rootRef is?

File: src/index.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 const RedisConnector = require('./connector/RedisConnector');
+const FirebaseConnector = require('./connector/FirebaseConnector');
 
 module.exports = {
     Multivariate: Multivariate,
     Connector: Connector,
-    RedisConnector: RedisConnector
+    RedisConnector: RedisConnector,
+    FirebaseConnector: FirebaseConnector
 }
```
Comment (anna-tsulun) [inline src/index.js:11]: Can you please add an empty line?

**Review by anna-tsulun — CHANGES_REQUESTED** [review]

File: src/connector/FirebaseConnector.js (hunk 1 above)
Comment (pamelazagatti) [reply src/connector/FirebaseConnector.js:5]: This is the name of the root key of the firebase database. In our case, we are passing the **experiments_stats** value inside of our node application.

**Pull request merged**